lib/wallaby/node.ex

```diff
@@ -238,10 +238,16 @@ defmodule Wallaby.Node do
   @doc """
   Matches the Node's content with the provided text.
   """
-  @spec has_content?(t, String.t) :: boolean()
+  @spec has_text?(t, String.t) :: boolean()
 
-  def has_content?(%Node{}=node, text) when is_binary(text) do
-    text(node) == text
+  def has_text?(%Node{}=node, text) when is_binary(text) do
+    retry fn ->
+      regex_results = Regex.run(~r/#{text}/, text(node))
+      if regex_results |> is_nil do
+        raise Wallaby.ExpectationNotMet, "Text '#{text}' not found"
+      end
+      true
```

Comment:
I think this might have gotten changed during the rebase. Shouldn't this be `regex_results`?